src/components/PublicHeader.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { mount } from '@cypress/react';
+
+import PublicHeader from './PublicHeader';
+
+describe('PublicHeader', () => {
+
+    const userData = {
```

Comment:
I would call this something along the lines of `dummyUser` or `mockUser` to indicate that it's not a real user object.